dashboard/db/migrate/20160213001925_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,5 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+  end
```

Comment:
Do we have any idea how long this migration will take in prod? There are approximately 422M rows (SELECT MAX(id) - MIN(id) FROM user_levels;).